adoc/deployment-preparation.adoc

```diff
@@ -172,8 +172,9 @@ as various default configurations and examples.
 .Using a Proxy Server
 ====
 
-Sometimes, you need a proxy server to be able to connects the SUSE Customer Center.
-To do that:
+Sometimes, you need a proxy server to be able to connect to the SUSE Customer Center.
```

Comment:
You can replace `SUSE Customer Center` with the `{scc}` entity.

Reply:
```suggestion
Sometimes, you need a proxy server to be able to connect to the {scc}.
```
Like that?